packages/sdk/src/Confidence.ts

```diff
@@ -247,13 +259,16 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
       id: SdkId.SDK_ID_JS_CONFIDENCE,
       version: '0.0.5', // x-release-please-version
     } as const;
-    const flagResolverClient = new FlagResolverClient({
+    let flagResolverClient: FlagResolverClient = new FetchingFlagResolverClient({
       clientSecret,
       fetchImplementation,
       sdk,
       environment,
       region,
     });
+    if (environment === 'client') {
+      flagResolverClient = new CachingFlagResolverClient(flagResolverClient, 3000_000);
```

Comment:
I think there's one or two 0s too much here? 3 thousand seconds?
Also, did we decide to extract this to be a setting? (can defer to a next release)